### DISCUSSION TITLE
Search by label using logical OR

### DISCUSSION THREAD

**Discussion by superkhau** [community/community#4507] [Projects and Issues] [answered]
Based on all the feedback received at https://github.com/isaacs/github/issues/660 over the last 5 years, I would like to keep the conversation going. The original content of the post was:

>I want to filter labels using a query like label:bug OR label:feature. Is there a way to do this? Seems to only support AND ATM.

**Comment by Levi-Lesches** [comment]
Note that the original issue has: 
- 672 👍 
- 59 🎉 
- 90 ❤️ 
- 30 👀

**Comment by dipree** [comment, marked as answer]
Hi everyone 👋, you can now search issues by label using logical OR. Simply use a comma to separate the labels. For example `label:"good first issue",bug` will list all issues with a label `good first issue` **or** `bug`.  See https://github.blog/changelog/2021-08-02-search-issues-by-label-using-logical-or/ .

**Comment by TomaszGasior** [comment]
This works in pull requests and issues lists in the repository. Unfortunately, this feature does not work in generic view for PRs/issues like https://github.com/pulls or https://github.com/issues. When I try to use it there, `label:"good first issue",bug` is converted to non-working `label:"good first issue,bug"`.

**Comment by lycarter** [comment]
Could this be added to the documentation? https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax does not mention this feature, nor is it exposed or mentioned in advanced search

**Reply by IndigoLily** [reply]
It's not documented on that page because it's not actually part of the search syntax, it's specifically a feature of the `label` qualifier (which is ridiculous imho), and it *is* documented [there](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-label).

**Comment by wrzasa** [comment]
OK, works for labels. That was the initial ask. But in the thread thread there was a lot of suggestions regarding this kind of feature for the other attributes (title, user, body etc) which seems logical. Will these be covered too?

**Comment by kingbuzzman** [comment]
What about other keys other than `label`? such as `author`?

**Comment by SarcevicAntonio** [comment]
Id love to be able to serach for `is:open is:pr (user-review-requested:@me OR (author:@me review:required))` so I can easily find everything that needs my attention ATM.

**Reply by aqeelat** [reply]
@dipree Is this on your radar?

**Comment by rileybroughten** [comment]
For those still following along here, wanted to flag this discussion for the [private beta of advanced search for Issues](https://github.com/orgs/community/discussions/137644), including `AND`/`OR` and nested searches. Follow along there as we'll be posting updates on when this is being expanded and how to join!

**Comment by jvandort** [comment]
I noticed this was [introduced](https://github.blog/developer-skills/application-development/github-issues-search-now-supports-nested-queries-and-boolean-operators-heres-how-we-rebuilt-it/) to the issues page. Its great to see this sort of functionality being implemented. 

However, I would love for this to be available for GitHub Projects. I think functionality like this is most useful in Projects, where I want to be able to build complex queries that are _persistent_. Having this sort of functionality on the issues page is nice but I only use that page when searching for issues adhoc. Projects is really where I want to be able to use complex queries.
